src/FunctionInfo.cs

```diff
@@ -30,7 +31,9 @@ internal class AzFunctionInfo
         internal readonly string FuncName;
         internal readonly string EntryPoint;
         internal readonly string ScriptPath;
+        internal readonly string DeployedPSFuncName;
```

Comment:
What's the difference between this and `FuncName` on line 31

Reply:
> DeployedPSFuncName = $"_{FuncName}_";

This is to avoid unexpected conflicts, which should be rare, but possible to happen.